plugin/embLayerNormPlugin/embLayerNormPluginLegacy.cpp

```diff
@@ -167,7 +167,8 @@ DimsExprs EmbLayerNormPluginDynamicLegacy::getOutputDimensions(
 
         // this code must match getMHAMaskPackedSize in bertCommon.h
         bool const isSmOK
-            = (mSM == kSM_75 || mSM == kSM_80 || mSM == kSM_86 || mSM == kSM_87 || mSM == kSM_89 || mSM == kSM_90);
+            = (mSM == kSM_75 || mSM == kSM_80 || mSM == kSM_86 || mSM == kSM_87 || mSM == kSM_89 || mSM == kSM_90
+             );
```

Comment:
Why the introduction of a newline here? Was this done by a formatter? Same question for the change in `plugin/embLayerNormPlugin/embLayerNormPlugin.cpp`.